paasta_tools/cli/cmds/status.py

```diff
@@ -70,12 +70,13 @@
 from paasta_tools.utils import PaastaColors
 from paasta_tools.utils import SystemPaastaConfig
 
+
 HTTP_ONLY_INSTANCE_CONFIG: Sequence[Type[InstanceConfig]] = [
     FlinkDeploymentConfig,
     KubernetesDeploymentConfig,
     AdhocJobConfig,
 ]
-SSH_ONLY_INSTANCE_CONFIG = [ChronosJobConfig]
+SSH_ONLY_INSTANCE_CONFIG: Sequence[Type[InstanceConfig]] = [ChronosJobConfig]
```

Comment:
I thought this list should be empty? Since Chronos can go either way now.

Reply:
so when it's empty, with the original logic, we use ssh even with use_api_enpoint being true.

should I rm it from the list?